packages/manager/src/features/ObjectStorage/AccessKeyLanding/AccessKeyRegions/AccessKeyRegions.tsx

```diff
@@ -29,13 +29,15 @@ export const AccessKeyRegions = (props: Props) => {
 
   return (
     <RegionMultiSelect
-      onChange={onChange}
+      placeholder={
+        selectedRegion.length > 0 ? ' ' : 'Select regions or type to search'
```

Comment:
Is there a reason for this space?

```suggestion
        selectedRegion.length > 0 ? '' : 'Select regions or type to search'
```

Reply:
Ah, fixed with 6df361f